<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-285KDFR" failures="3" tests="15" name="Sprint1" time="0.333" errors="0" timestamp="2022-10-10T22:18:16 IST">
  <testcase classname="rerunTestCases.TestCases" name="loginWhatsApp" time="0.035">
    <skipped/>
  </testcase> <!-- loginWhatsApp -->
  <testcase classname="rerunTestCases.TestCases" name="loginWhatsApp" time="0.0">
    <skipped/>
  </testcase> <!-- loginWhatsApp -->
  <testcase classname="rerunTestCases.TestCases" name="loginWhatsApp" time="0.0">
    <skipped/>
  </testcase> <!-- loginWhatsApp -->
  <testcase classname="rerunTestCases.TestCases" name="loginWhatsApp" time="0.0">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at rerunTestCases.TestCases.loginWhatsApp(TestCases.java:17)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- loginWhatsApp -->
  <testcase classname="rerunTestCases.TestCases" name="loginFB" time="0.0"/>
  <testcase classname="rerunTestCases.TestCases" name="loginTwitter" time="0.0"/>
  <testcase classname="rerunTestCases.TestCases2" name="loginAmazon" time="0.0">
    <skipped/>
  </testcase> <!-- loginAmazon -->
  <testcase classname="rerunTestCases.TestCases2" name="loginAmazon" time="0.016">
    <skipped/>
  </testcase> <!-- loginAmazon -->
  <testcase classname="rerunTestCases.TestCases2" name="loginAmazon" time="0.0">
    <skipped/>
  </testcase> <!-- loginAmazon -->
  <testcase classname="rerunTestCases.TestCases2" name="loginAmazon" time="0.0">
    <failure type="java.lang.ArithmeticException" message="/ by zero">
      <![CDATA[java.lang.ArithmeticException: / by zero
at rerunTestCases.TestCases2.loginAmazon(TestCases2.java:10)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- loginAmazon -->
  <testcase classname="rerunTestCases.TestCases2" name="loginFlipkart" time="0.016"/>
  <testcase classname="rerunTestCases.TestCases2" name="loginMyntra" time="0.015">
    <skipped/>
  </testcase> <!-- loginMyntra -->
  <testcase classname="rerunTestCases.TestCases2" name="loginMyntra" time="0.0">
    <skipped/>
  </testcase> <!-- loginMyntra -->
  <testcase classname="rerunTestCases.TestCases2" name="loginMyntra" time="0.016">
    <skipped/>
  </testcase> <!-- loginMyntra -->
  <testcase classname="rerunTestCases.TestCases2" name="loginMyntra" time="0.015">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at rerunTestCases.TestCases2.loginMyntra(TestCases2.java:19)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- loginMyntra -->
</testsuite> <!-- Sprint1 -->
